lib/ohloh_scm/adapters/bzr/misc.rb

```diff
@@ -38,5 +38,12 @@ def export(dest_dir, token=head_token)
 
 			run "cd '#{url}' && bzr export --format=dir -r #{to_rev_param(token)} '#{dest_dir}'"
 		end
+
+    def tags
+      tag_strings = run("cd '#{url}' && bzr tags").split(/\n/)
+      tag_strings.map do |tag_string|
```

Comment:
`each_line` would not return an array like `map` would. We could do `each_line.map`, but `each_line` would retain the `\n` in the results. So it would need an additional `chomp` anyways.
```ruby
>> "one\ntwo".each_line.first
"one\n"
```